packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/options/options.html

```diff
@@ -13,6 +13,7 @@
         >
             <oui-tile-description>
                 <div
+                    class="no-break-word"
```

Comment:
Possibility use default bootstrap classes ? 

`text-break` or `test-nowrap` ?

https://getbootstrap.com/docs/4.4/utilities/text/#text-wrapping-and-overflow